### PR TITLE
Updating ceph-iscsi docs and deployment

### PR DESCRIPTION
**Pull request by asettle**
This PR also contains a placeholder for information on what
IP addresses to apply.

This PR should also have in scope further information on
ceph-iscsi and how to operate with cephadm.

### CODE REVIEW

File: xml/deploy_cephadm.xml [hunk 1]
```diff
@@ -1435,7 +1443,11 @@ spec:
   pool: iscsi-pool
   api_user: iscsi-user
   api_password: password
+  trusted_ip_list: "<replaceable>IP_ADDRESS</replaceable>, <replaceable>IP_ADDRESS</replaceable>"
```
Comment (ricardoasmarques) [inline xml/deploy_cephadm.xml:1446]: `IP_ADDRESS_1, IP_ADDRESS_2` instead of `IP_ADDRESS, IP_ADDRESS`?

Comment (asettle) [reply xml/deploy_cephadm.xml:1446]: The presumption is the reader is smart enough to assertain to include the same IP address twice?

**Review by ricardoasmarques — APPROVED** [review]
lgtm

File: xml/deployment_iscsi.xml [hunk 2]
```diff
@@ -0,0 +1,824 @@
...
+       <para>
+        If set to 1, turns on Write Cache Enable.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>enforce_pr_isids</term>
+      <listitem>
+       <para>
+        If set to 1, enforces persistent reservation ISIDs.
```
Comment (lproven) [inline xml/deployment_iscsi.xml:680]:
```suggestion
        If set to <literal>1</literal>, enforces persistent reservation ISIDs.
```

**Review by lproven — CHANGES_REQUESTED** [review]
WIP review -- more to come ASAP

**Comment by asettle** [comment]
> WIP review -- more to come ASAP

Will apply in a new patch :) thank you @lproven !

File: xml/deployment_iscsi.xml [hunk 3]
```diff
@@ -0,0 +1,824 @@
...
+   <tip>
+    <para>
+     You can use the <command>ls</command> command to review the configuration.
+     Some configuration nodes also support the <command>info</command> command,
+     which can be used to display more detailed information.
+    </para>
+   </tip>
+   <para>
+    Note that, by default, ACL authentication is enabled so this target is not
+    accessible yet. Check <xref linkend="iscsi-lrbd-autentication" /> for more
```
[the remark this answers is not in the document]
Comment (asettle) [reply xml/deployment_iscsi.xml:357]: fixed

File: xml/deployment_iscsi.xml [hunk 4]
```diff
@@ -0,0 +1,824 @@
...
+  This chapter introduces detailed information to set up a &ceph; cluster
+  infrastructure together with an iSCSI gateway so that the client hosts can
+  use remotely stored data as local storage devices using the iSCSI protocol.
+ </para>
+ <sect1 xml:id="ceph-iscsi-iscsi">
+  <title>iSCSI Block Storage</title>
+
+  <para>
+   iSCSI is an implementation of the Small Computer System Interface (SCSI)
+   command set using the Internet Protocol (IP), specified in RFC 3720. iSCSI
```
[the remark this answers is not in the document]
Comment (smithfarm) [reply xml/deployment_iscsi.xml:49]: I defer to the experts @ricardoasmarques and @ddiss

**Pull request merged**